requirements.txt

```diff
@@ -5,4 +5,5 @@ pytest==3.6.4
 pytest-ordering==0.5
 funcsigs==1.0.2
 mock==2.0.0
+selinux
```

Comment:
NO this is not ok, what you want is the Makefile with 'install-deps' target. Like this: https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/systemfacts/Makefile

Reply:
@vinzenz , I put libselinux-python to Makefile, still that hasn't help to the import.